cumulus/zombienet/tests/assign-core.js

```diff
@@ -0,0 +1,46 @@
+// Assign a parachain to a core.
```

Comment:
could we reuse the assign-core script from the polkadot folder? maybe move it to some common zombienet helper scripts folder in the root folder